odoo/addons/base/models/ir_qweb.py

```diff
@@ -192,7 +192,7 @@ def _compile_directive_call_assets(self, el, options, indent):
 
         code = self._flushText(options, indent)
         code.append(self._indent(dedent("""
-            t_call_assets_nodes = self._get_asset_nodes(%(xmlid)s, compile_options, css=%(css)s, js=%(js)s, debug=values.get("debug"), async_load=%(async_load)s, defer_load=%(defer_load)s, lazy_load=%(lazy_load)s, media=%(media)s)
+            t_call_assets_nodes = self._get_asset_nodes(%(xmlid)s, css=%(css)s, js=%(js)s, debug=values.get("debug"), async_load=%(async_load)s, defer_load=%(defer_load)s, lazy_load=%(lazy_load)s, media=%(media)s)
```

Comment:
@Gorash can you check that

Reply:
Looks good to me. :+1: 
Effectively options in `_get_asset_content` were no longer useful and therefore the options parameters either.